### ISSUE TITLE
As a user I want a slanting tool

### ISSUE THREAD

**Issue by nilsschmidt1337** [nilsschmidt1337/ldparteditor#614]
## Initial dialog text:

A tool that generates and applies a projection matrix from 2 or 3 lines to any selection.

There are two possibilies to use this tool:

a) Two-line variant
1. Create two lines with one common vertex.
  Example A: (red line = X axis, blue line = Z axis)
    2 4 0 0 0 1 0 0
    2 1 0 0 0 0 0 1
  Example B: (red line = X axis, green line = Y axis)
    2 4 0 0 0 1 0 0
    2 2 0 0 0 0 1 0

2. Select only these two lines in the 3D editor (LDPE will automatically calculate the matrix)
3. Select the element(s) which should be changed and open this tool again.

a) Three-line variant

1. Create three lines with one common vertex. One red line for the X axis, one green line for the Y axis, and one blue line for the Z axis.
  Example:
    2 4 0 0 0 1 0 0
    2 2 0 0 0 0 1 0
    2 1 0 0 0 0 0 1

2. Select only these three lines in the 3D editor (LDPE will automatically calculate the matrix)
3. Select the element(s) which should be changed and open this tool again.

## Dialog after successful selection

- [x] Set origin to center of the first axis pair (useful for rect-primitives)

The following matrix will be applied to your selection.
```
a b c x
d e f y
g h i z
0 0 0 1
```

**OR**

You haven't selected anything, but the following matrix is calculated based on 2 **OR** 3 axes.
```
a b c x
d e f y
g h i z
0 0 0 1
```

**Comment by nilsschmidt1337** [comment]
I just need to do more testing. Then it is ready to go! 😁

**Comment by nilsschmidt1337** [comment]
Closed with commit ff5ec0a.
